src/test/java/PageObjects/BasePage.java

```diff
@@ -19,7 +21,14 @@ public BasePage() {
         httpRequest = RestAssured.given();
 
     }
-
+    public void assertStatusCode(int statusCode){
+        Assertions.assertEquals(statusCode, response.getStatusCode());
```

Comment:
Really good for reusability! Just wanted to check with Matt if this is best practice